### ISSUE TITLE
Debian package hashes

### ISSUE THREAD

**Issue by szepeviktor** [stintel/vallumd#10]
The md5sum file has invalid (build-time) paths.

**Comment by stintel** [comment]
Can you provide some details? The package generation is done with CMake/CPack, so I suspect this could be a bug in the version you use.

**Comment by szepeviktor** [comment]
I use Debian version 3.7.2-1.
The md5sums contains
```
fb12e782be0e355f4b61654f6261dab3  /root/dist-mod/vallumd/_CPack_Packages/Linux/DEB/vallumd-0.2.0-0-pre-39-g4c71699-1_amd64/usr/sbin/vallumd
f159b983905f98dca09662b0a55db9ec  /root/dist-mod/vallumd/_CPack_Packages/Linux/DEB/vallumd-0.2.0-0-pre-39-g4c71699-1_amd64/etc/default/vallumd
```

**Comment by stintel** [comment]
Looks like this is fixed in CMake 3.8: https://cmake.org/pipermail/cmake/2017-March/065109.html
Consider asking your distribution to backport the fix.

**Comment by szepeviktor** [comment]
Thanks.
https://gitlab.kitware.com/cmake/cmake/issues/16517
